doc/reference/configuration/configuration_reference.rst

```diff
@@ -2427,9 +2500,19 @@ The ``process`` defines configuration parameters of the Tarantool process in the
 
 .. confval:: process.title
 
-    Sets the name of the server's process title (what’s shown in the COMMAND column for
-    ``ps -ef`` and ``top -c`` commands).
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    Add the given string to the server's process title
+    (it is shown in the COMMAND column for the Linux commands
+    ``ps -ef`` and ``top -c``).
+
+    For example, if you set ``title: 'myservice - instance1'``,
+    :samp:`ps -ef` shows the Tarantool server process like this:
```

Comment:
I suggest to use `title: myservice - {{ instance_name }}` instead of `title: myservice - instance1`, because it works for all the instance names. See the [predefined variables](https://www.tarantool.io/en/doc/latest/concepts/configuration/#predefined-variables) section.

Maybe it is better to show the option example as a separate block, don't inline it into the prose text.

```yaml
process:
  title: myservice - {{ instance_name }}
```